catalogue/webapp/pages/item.js

```diff
@@ -186,13 +187,12 @@ const IIIFCanvasThumbnail = ({
   const urlTemplate = iiifImageTemplate(thumbnailService['@id']);
   return (
     // TODO: add alt text
-    <img
+    <ResponsiveImage
+      alt=""
       width={size.width}
       height={size.height}
-      src={urlTemplate({
-        size: `${size.width},${size.height}`,
-      })}
-      alt=""
+      url={urlTemplate({ size: `${size.width},${size.height}` })}
```

Comment:
Could we not make this a IIIFResponsiveImage (as it relies on the IIIF Image API) and just send the template down the pipes?

Reply:
Saying this - nearly all our images are from Loris, so follow this pattern, but is there a harm in being explicit about it?